app/models/ems_refresh/save_inventory_cinder.rb

```diff
@@ -4,15 +4,7 @@
 #   - cloud_volumes
 #   - cloud_volume_backups
 #   - cloud_volume_snapshots
-#   - vms
-#     - storages (link)
-#     - security_groups (link)
-#     - operating_system
-#     - hardware
-#       - disks
-#       - guest_devices
-#     - custom_attributes
-#     - snapshots
+#   - backing_links
 #
 
 module EmsRefresh::SaveInventoryCinder
```

Comment:
@agrare As it stands now, given the variability between various storage managers, we can't really assume there will be much commonality between the save methods. After we add a few more storage managers of various types, we may  be able to identify patterns and refactor.